src/webapp/azext_webapp/create_util.py

```diff
@@ -91,3 +102,78 @@ def check_app_exists(cmd, rg_name, app_name):
         if item.name == app_name:
             return True
     return False
+
+
+def get_lang_from_content(src_path):
+    # NODE: package.json should exisit in the application root dir
```

Comment:
typo: exisit

Reply:
:) thanks @derekbekoe . i will have this fixed in my next PR